src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -94,14 +94,51 @@ int halide_hexagon_remote_load_library(const char *soname, int sonameLen,
 
 volatile int power_ref_count = 0;
 
+// HAP power_context
+//  - has to be non-NULL (for devices newer than SM8350)
+//  - only has to be unique within the process
+//  - the context is just an identifier
+//  - even though it is a void*, it is never stored into
+//  - for managing multiple clients, could just malloc(1)/free contexts
+//  - could even just use "(void*)1" (non-NULL), but that would not be
```

Comment:
They are asking us not to use a constant as there could be a problem if the app had multiple/different uses of Hexagon (e.g. Halide, ML, ...) and was linked with multiple/different skel files....each of which the power team wants to see using a different power context.  If every skel file just used a constant '1', then they couldn't distinguish the different Hexagon clients requests from each other.

Reply:
Instead of using the address of a writable variable as the power context, we could instead use the read-only address of one of the other functions in halide_remote.cpp: 

```
static inline void *get_HAP_power_context() {
    // Halide runtime only needs a single power context
    return (void*)halide_hexagon_remote_load_library;
}
```

Reply:
I see, using 1 could accidentally collide with another use. What you have in the PR currently is fine. I think this is ready to merge now.